pkg/apigw/ctx/ctx.go

```diff
@@ -3,12 +3,14 @@ package ctx
 import (
 	"context"
 
+	"github.com/cortezaproject/corteza-server/pkg/apigw/profiler"
 	"github.com/cortezaproject/corteza-server/pkg/apigw/types"
 )
 
 type ContextKey string
 
 const ContextKeyScope ContextKey = "scope"
+const ContextKeyProfiler ContextKey = "profiler"
```

Comment:
can you change this (both?) to follow "empty-struct" approach (see `identityCtxKey`) we use for context key?

Reply:
:+1: agree